code/game/machinery/vending_vr.dm

```diff
@@ -1216,7 +1216,7 @@
 					/obj/item/clothing/suit/imperium_monk = 2000,
 					/obj/item/clothing/suit/holidaypriest = 200,
 					/obj/item/clothing/head/witchwig = 200,
-					/obj/item/clothing/under/sundress = 200,
+					/obj/item/clothing/under/sundress = 50,
```

Comment:
Same here.

Reply:
Uniform item, so i think it should be 50 IMO